src/Hedgehog.Experimental/Hedgehog.Experimental.fsproj

```diff
@@ -1,4 +1,4 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
What's the change here? I can't see any difference.

Reply:
Ah - my git client hid this from me and I didn't check the diff in GitHub >_> Fixed.

Reply:
Thanks. Out of interest, do you have any idea what the change actually was? The lines looked identical to me.

Reply:
Not sure - I'm guessing a `\r\n` got changed to an `\r` or something. Not sure why it does this. I've seen this before when updating packages with Visual Studio `fsprojs`s, so it's consistent.